lib/config/default.js

```diff
@@ -1256,6 +1256,66 @@ defaultConfig.definition = () => ({
         default: ''
       }
     }
+  },
+  /**
+   * Security agent configurations
+   */
+  security: {
+    /**
+     * Enables/Disables security agent functions and generation of events.
+     */
+    enabled: {
+      formatter: boolean,
+      default: false
+    },
+
+    /**
+     * To completely disable security, set security.agent.enabled to false. This property is read only once at application start.
+     */
+    agent: {
+      enabled: {
+        formatter: boolean,
+        default: false
+      }
+    },
+
+    /**
+     *  NR security provides two modes IAST and RASP. Default is IAST
+     */
+    mode: {
+      default: 'IAST'
+    },
+
+    /**
+     * Security validator URL
+     */
+    validator_service_url: {
+      default: 'wss://csec.nr-data.net'
+    },
+
+    /**
+     * Following category of security events can be disabled from generating.
```

Comment:
Do we need to clarify this? I'm not immediately sure what's meant.

Reply:
I don't know. I copied this from their clone